feature/interface/aggregate/otg_tests/aggregate_all_not_viable_test/aggregate_all_not_forwarding_viable_test.go

```diff
@@ -0,0 +1,1026 @@
+		dutPortList = append(dutPortList, dut.Port(t, fmt.Sprintf("port%d", portIdx+1)))
+	case LAG2:
+		for portIdx < a.ateLagCount {
+			portList = append(portList, dut.Port(t, fmt.Sprintf("port%d", portIdx+2)))
+			dutPortList = append(dutPortList, dut.Port(t, fmt.Sprintf("port%d", portIdx+2)))
+			portIdx++
+		}
+	case LAG3:
+		for portIdx < a.ateLagCount {
+			portList = append(portList, dut.Port(t, fmt.Sprintf("port%d", portIdx+7)))
+			dutPortList = append(dutPortList, dut.Port(t, fmt.Sprintf("port%d", portIdx+7)))
+			portIdx++
+		}
+	}
+	return portList
+}
+
+// configDstAggregateDUT configures port-channel destination ports
+func configAggregateDUT(dut *ondatra.DUTDevice, i *oc.Interface, a *aggPortData) {
+	i.Description = ygot.String(a.ateAggName)
+	if deviations.InterfaceEnabled(dut) {
+		i.Enabled = ygot.Bool(true)
+	}
+	s := i.GetOrCreateSubinterface(0)
+	s4 := s.GetOrCreateIpv4()
+	if deviations.InterfaceEnabled(dut) {
+		s4.Enabled = ygot.Bool(true)
+	}
+	a4 := s4.GetOrCreateAddress(a.dutIPv4)
+	a4.PrefixLength = ygot.Uint8(ipv4PLen)
+
+	s6 := s.GetOrCreateIpv6()
+	if deviations.InterfaceEnabled(dut) {
+		s6.Enabled = ygot.Bool(true)
+	}
+	s6.GetOrCreateAddress(a.dutIPv6).PrefixLength = ygot.Uint8(ipv6PLen)
+	i.Type = ieee8023adLag
+	g := i.GetOrCreateAggregation()
+	g.LagType = lagTypeLACP
+}
+
+// setupAggregateAtomically setup port-channel based on LAG type.
+func setupAggregateAtomically(t *testing.T, dut *ondatra.DUTDevice, aggID string, agg *aggPortData, portList []*ondatra.Port) {
+	d := &oc.Root{}
+	d.GetOrCreateLacp().GetOrCreateInterface(aggID)
+
+	aggr := d.GetOrCreateInterface(aggID)
+	aggr.GetOrCreateAggregation().LagType = oc.IfAggregate_AggregationType_LACP
+	aggr.Type = oc.IETFInterfaces_InterfaceType_ieee8023adLag
+
+	for _, port := range portList {
+		i := d.GetOrCreateInterface(port.Name())
+		i.GetOrCreateEthernet().AggregateId = ygot.String(aggID)
+
+		i.Type = oc.IETFInterfaces_InterfaceType_ethernetCsmacd
+
+		if deviations.InterfaceEnabled(dut) {
+			i.Enabled = ygot.Bool(true)
+		}
+	}
+	p := gnmi.OC()
+	fptest.LogQuery(t, fmt.Sprintf("%s to Update()", dut), p.Config(), d)
+	gnmi.Update(t, dut, p.Config(), d)
+}
+
+// clearAggregate delete any previously existing members of aggregate.
+func clearAggregate(t *testing.T, dut *ondatra.DUTDevice, aggID string, agg *aggPortData, portList []*ondatra.Port) {
+	// Clear the aggregate minlink.
+	gnmi.Delete(t, dut, gnmi.OC().Interface(aggID).Aggregation().MinLinks().Config())
+	// Clear the members of the aggregate.
+	for _, port := range portList {
+		resetBatch := &gnmi.SetBatch{}
+		gnmi.BatchDelete(resetBatch, gnmi.OC().Interface(port.Name()).Ethernet().AggregateId().Config())
+		gnmi.BatchDelete(resetBatch, gnmi.OC().Interface(port.Name()).ForwardingViable().Config())
+		resetBatch.Set(t, dut)
+	}
+}
+
+// configureDUTBGP configure BGP on DUT
+func configureDUTBGP(t *testing.T, dut *ondatra.DUTDevice, aggIDs []string) {
```

Comment:
Should use the configplugin for bgp instead:

https://github.com/openconfig/featureprofiles/blob/main/internal/cfgplugins/bgp.go

Reply:
Looks like the above package is for EBGP not iBGP